packages/js/src/import.js

```diff
@@ -0,0 +1,144 @@
+import jQuery from "jquery";
+
+import IndexingService from "./services/IndexingService";
+
+const AioseoV4 = "WPSEO_Import_AIOSEO_V4";
+
+let importButton, importForm, spinner, loading_msg, checkMark, errorMark;
+
+/**
+ * Adds Progress UI elements in the page.
+ *
+ * @returns {void}
+ */
+function addProgressElements() {
+	jQuery( spinner ).insertAfter( importButton );
+	jQuery( checkMark ).insertAfter( spinner );
+	jQuery( errorMark ).insertAfter( spinner );
+	jQuery( loading_msg ).insertAfter( spinner );
+}
+
+/**
+ * Function called when importing progress is made.
+ *
+ * @param {number} count The amount of items processed.
+ *
+ * @returns {void}
+ */
+function importingProgress( count ) {
+	console.log( "Now processed", count, "items" );
+	spinner.show();
+	loading_msg.show();
+
+	importButton.prop( "disabled", true );
+}
+
+/**
+ * Function called when importing is completed succesfully.
+ *
+ * @returns {void}
+ */
+function importingSuccess() {
```

Comment:
Optionally we could simply remove the AIOSEO `option` element from the select here as well.